core/modules/config/config.admin.inc

```diff
@@ -18,7 +18,7 @@ function config_sync_form(array $form, array &$form_state) {
   $form['#attached']['css'][] = backdrop_get_path('module', 'config') . '/css/config.admin.css';
 
   $form['description'] = array(
-    '#markup' => '<p>' . t('This form allows you to synchronize configuration between sites, such as in development, staging, and production servers. This page shows you the current differences between your current configuration and changes that have been staged for deployment.') . '</p>',
+    '#markup' => '<p>' . t('This form allows you to synchronize configuration between sites, such as in development, staging, and production servers. This page shows you the current differences between your current configuration and changes that have been staged for deployment.') . '</p></p>' . t('For some development workflows (such as when the configuration files are in version control) it may be necessary to not delete the staging config files after each configuration sync. The <code>config_sync_clear_staging</code> setting can be disabled in <code>settings.php</code>. Read more about <a href="!url" target="_blank">managing configuration</a>.', array('!url' => url('https://api.backdropcms.org/documentation/working-with-configuration'))) . '</p>',
```

Comment:
This new section is quite verbose... taking a stab at an alternative.

When configuration files are in version control it may be preferable not to empty the staging directory after each sync. The <code>config_sync_clear_staging</code> setting can be disabled in <code>settings.php</code>

Reply:
There's also some pre-existing duplication of the word "current" in `...the current differences between your current configuration...`. How about this?:

> This form allows you to synchronize configuration between sites, such as in development, staging, and production servers. Any differences between your current configuration and the changes that have been staged for deployment will be listed below.

@jenlampton I do like the use of `may be preferable` instead of `may be necessary` 👍 ...and there should be a comma in there, for readability:

> When configuration files are in version control, it may be preferable not to empty the staging directory after each sync. The `config_sync_clear_staging` setting can be disabled in `settings.php`.